Update localizing stores.md (#957)
bigcommerce/dev-docs: docs/stencil-docs/localization/localizing-stores.md

<div class="otp" id="no-index">

### On this page
- [Localizing theme files](#localizing-theme-files)
- [Localizing checkout](#localizing-checkout)
- [Localizing control panel content](#localizing-control-panel-content)
- [Resources](#resources)

</div>

> A BigCommerce storefront can be customized to display in any one language of your choice. To fully localize a store into a language/region, you will need to customize three areas:

Reviewer: ```suggestion
A BigCommerce storefront can be customized to display in any one language of your choice. To fully localize a store for a language or region, you will need to customize three areas:

```

</div>
</div>

## Localizing theme files

> 

Reviewer: @traci707, can you include an introductory sentence here. We want to avoid empty headers that only contain other headers.

### Translating theme files by Stencil CLI

The bulk of localizing a store is translating content on your theme's page templates. You will need to abstract these templates' literal text strings into `{{lang}}` variables, then provide per-language JSON files that define these variables in corresponding key/value pairs.

### Translating strings by BigCommerce

> Some of the strings displayed on your storefront are not currently editable in BigCommerce. These are strings such as "This promotion cannot be applied with the selected currency." For strings such as this BigCommerce, will automatically render the strings in the shopper's language as defined by the shopper's browser language setting.

Reviewer 2: ```suggestion
Some of the strings displayed on your storefront are not currently editable in BigCommerce. These are strings such as "This promotion cannot be applied with the selected currency." BigCommerce will automatically render such strings in the shopper's language as defined by the shopper's browser language settings.

```

> 

Reviewer 2: ```suggestion
It may be best to list the languages alphabetically.

* Dutch
* English
* French
* German
* Italian
* Portuguese
* Spanish

```

Currently BigCommerce supports these uneditable strings in the following languages:
* English
* French
* German
* Spanish
* Italian
* Dutch
* Portuguese

> If you would like to review these strings, you can do so by downloading the language files we have made available in the .po file format. We recommend viewing these in a text editor designed to handle the .po file format or use a simple text editor.

Reviewer 2: ```suggestion
You can review these strings by downloading the language files we have made available in the portable object format. We recommend viewing these in a simple text editor or a text editor designed to handle PO files.
```

* [Multi-Language Checkout](https://developer.bigcommerce.com/stencil-docs/localization/multi-language-checkout)
* [Adding Products](https://support.bigcommerce.com/s/article/Adding-Products-v3) (BigCommerce Knowledge Base)
* [Changing Your Default Currency](https://support.bigcommerce.com/s/article/Managing-Currencies#default)(BigCommerce Knowledge Base)

### Additional resources

* [checkout.html](https://github.com/bigcommerce/cornerstone/blob/master/templates/pages/checkout.html) (BigCommerce GitHub)
* [order-confirmation.html](https://github.com/bigcommerce/cornerstone/blob/master/templates/pages/order-confirmation.html) (BigCommerce GitHub)
* [JSON translation file](https://github.com/bigcommerce/cornerstone/tree/master/lang) (BigCommerce GitHub)

### Language files

* [Dutch](https://bigcommerce.github.io/dev-docs/assets/PO/storefront-nl-NL.po)
* [French](https://bigcommerce.github.io/dev-docs/assets/PO/storefront-fr-FR.po)
* [German](https://bigcommerce.github.io/dev-docs/assets/PO/storefront-de-DE.po)
* [Italian](https://bigcommerce.github.io/dev-docs/assets/PO/storefront-it-IT.po)

> * [Portuguese Brazil](https://bigcommerce.github.io/dev-docs/assets/PO/storefront-pt-BR.po)

Reviewer: ```suggestion
* [Portuguese (Brazil)](https://bigcommerce.github.io/dev-docs/assets/PO/storefront-pt-BR.po)

```
